Formula/arm-linux-gnueabihf-binutils.rb

```diff
@@ -0,0 +1,31 @@
+class ArmLinuxGnueabihfBinutils < Formula
+  desc "FSF/GNU binutils for cross-compiling to arm-linux"
+  homepage "https://www.gnu.org/software/binutils/binutils.html"
+  url "https://ftp.gnu.org/gnu/binutils/binutils-2.30.tar.gz"
+  mirror "https://ftpmirror.gnu.org/binutils/binutils-2.30.tar.gz"
+  sha256 "8c3850195d1c093d290a716e20ebcaa72eda32abf5e3d8611154b39cff79e9ea"
+
+  keg_only "clashes with other cross-compilation binutils"
+
+  def install
+    system "./configure", "--disable-debug",
+                          "--disable-dependency-tracking",
+                          "--enable-deterministic-archives",
+                          "--prefix=#{prefix}",
+                          "--infodir=#{info}",
```

Comment:
`--infodir` and `--mandir` are unnecessary, `--prefix` is enough

Reply:
Done